boot/freeldr/freeldr/lib/fs/ntfs.c

```diff
@@ -733,6 +733,7 @@ static BOOLEAN NtfsLookupFile(PNTFS_VOLUME_INFO Volume, PCSTR FileName, PNTFS_MF
             TRACE("- Failed\n");
             return FALSE;
         }
+        CurrentMFTIndex &= NTFS_MFT_MASK;
```

Comment:
Not necessary anymore, since `NtfsFindMftRecord` applies the mask (see PR #3775).
```suggestion
```